README.md

```diff
@@ -95,7 +95,16 @@ npm run package version
 
 Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com/getbootstrapcdn#backer)]
 
-[![Bakers](https://opencollective.com/getbootstrapcdn/backers.svg?width=890)](https://opencollective.com/getbootstrapcdn#backers)
```

Comment:
I'm unsure what's the preferred way to do this. The single image approach is simpler, obviously.